FireBaseNotificationService.java

```diff
@@ -1335,12 +1335,11 @@ public void sendPaymentNotification(String message, String paymentUrl) {
             builder = new NotificationCompat.Builder(context);
         }
         // NotificationManager notificationManager = (NotificationManager) context.getSystemService(NOTIFICATION_SERVICE);
-        Notification notification = builder.build();
-        notification.flags |= Notification.FLAG_AUTO_CANCEL;
-        builder.setSmallIcon(R.drawable.icon_app)
```

Comment:
the builder is being configured after it has already been used to setup the notification, so this part of the configuration never makes it into the notification. Without a small icon, you will get an IllegalArgumentException crash in https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/app/NotificationManager.java#399, which is the observed behaviour as well. So recommending this change in the code.